apps/dashboard/src/components/workflow-editor/workflow-editor-provider.tsx

```diff
@@ -228,30 +257,37 @@ export const WorkflowEditorProvider = ({ children }: { children: ReactNode }) =>
   return (
     <WorkflowEditorContext.Provider value={value}>
       <AlertDialog open={blocker.state === 'blocked'}>
-        <AlertDialogContent>
+        <AlertDialogContent className="w-96">
           <AlertDialogHeader className="flex flex-row items-start gap-4">
-            <div className="bg-warning/10 rounded-lg p-3">
-              <RiAlertFill className="text-warning size-6" />
+            <div
+              className={`rounded-lg p-3 transition-all duration-300 ${
+                !isPending ? 'bg-success/10 scale-110' : 'bg-warning/10'
+              }`}
+            >
+              <div className="transition-opacity duration-300">
+                {!isPending ? (
+                  <CheckCircleIcon className="text-success animate-in fade-in size-6" />
+                ) : (
+                  <RiAlertFill className="text-warning animate-in fade-in size-6" />
+                )}
+              </div>
             </div>
             <div className="space-y-1">
```

Comment:
Please ensure that there is no layout shift during the state transition and the dialog shape and dimensions remain the same.